test/index.test.js

```diff
@@ -201,6 +201,11 @@ describe('ol-mapbox-style', function() {
               'maxzoom': 12,
               'tileSize': 256,
               'tiles': ['https://ahocevar.com/geoserver/gwc/service/wms?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetMap&FORMAT=image/png&SRS=EPSG:900913&LAYERS=topp:states&STYLES=&WIDTH=256&HEIGHT=256&BBOX={bbox-epsg-3857}']
+            },
+            'hillshading': {
+              'type': 'raster',
+              'tileSize': 1024,
+              'url': 'https://api.tiles.mapbox.com/v3/mapbox.geography-class.json'
```

Comment:
Please avoid requests to the outside world. You could use a data uri as url, e.g.
```js
'hillshading': {
  'type': 'raster',
  'url': 'data:text/plain;charset=utf-8;' + JSON.stringify({
    'tiles': [
      'path/to/tiles/{z}/{x}/{y}.png'
    ]
  })
}
```

Reply:
The data url was not working: invalid url errors from the browser (xhr api).
I'ver create a new fixture instead.